source/Octo.Tests/Commands/CreateReleaseCommandFixture.cs

```diff
@@ -77,12 +79,11 @@ public void ShouldThrowForBadMachine()
             CommandLineArgs.Add("--apikey=API-test");
             CommandLineArgs.Add("--project=Test Project");
             CommandLineArgs.Add("--releaseNumber=1.0.0");
-            CommandLineArgs.Add("--specificmachines=bad");
+            CommandLineArgs.Add("--specificmachines=badMach");
             CommandLineArgs.Add($"--deployto={ValidEnvironment}");
-            Assert.ThrowsAsync<CommandException>(async delegate
-            {
-                await createReleaseCommand.Execute(CommandLineArgs.ToArray());
-            });
+
+            var ex = Assert.ThrowsAsync<CommandException>(() => createReleaseCommand.Execute(CommandLineArgs.ToArray()));
+            ex.Message.Should().Be("The following specific machines could not be found: badMach");
```

Comment:
Consider making these messages consistent... But... not critical.

Reply:
Done.